xen/xenops_interface.ml

```diff
@@ -774,6 +774,11 @@ module XenopsAPI (R : RPC) = struct
       in
       declare "PCI.list" []
         (debug_info_p @-> VM.vm_id_p @-> returning stat_list_p err)
+
+    let dequarantine =
+      let boolean = Param.mk Types.bool in
+      declare "PCI.dequarantine" []
+        (debug_info_p @-> pci_addr_p @-> returning boolean err)
```

Comment:
So what is the meaning of the returned Boolean? Can we add some doc string?

Reply:
Updated. The boolean indicates success.

Reply:
@edwintorok raised a good point: should use exceptions for the error handling. I'll update the interface.

Reply:
Do we need that, since we also have the `err`? This is used to cause an exception to be raised on the client in case of an error (exception raised in the server).

Reply:
Aha,  a commenting race condition ;)

Reply:
I meant that the return type can be `returning unit err`, and just raise the exception in the server if it'd be false.

Reply:
Yes, that's what I meant, too.